feature/metis/conversation/src/test/kotlin/de/tum/informatics/www1/artemis/native_app/feature/metis/conversation/ConversationAnswerMessagesE2eTest.kt

```diff
@@ -64,4 +64,47 @@ class ConversationAnswerMessagesE2eTest : ConversationMessagesBaseTest() {
             }
         }
     }
-}
+
+    @Test(timeout = DefaultTestTimeoutMillis)
+    fun `resolving a post by updating its answer message`() {
```

Comment:
Please adapt the test GIVEN WHEN THEN naming scheme for all method names in this class.